code/game/objects/items/devices/traitordevices.dm

```diff
@@ -1,22 +1,35 @@
 /obj/item/jammer
 	name = "radio jammer"
-	desc = "Device used to disrupt nearby radio communication."
+	desc = "Fog of war that fits your pocket. Flicking the switch and extending the antenna will scramble nearby radio comms, making outgoing messages hard to understand."
```

Comment:
Description is a bit wordy, but thats just my opinion

Reply:
yeah i tried to make it more precise, and not concise. i think brevity in stuff like this hurts the theme a lot.

Reply:
im gonna resolve this i think